src/js/tech/html5.js

```diff
@@ -74,6 +74,7 @@ class Html5 extends Tech {
       this.handleTextTrackChange_ = Fn.bind(this, this.handleTextTrackChange);
       this.handleTextTrackAdd_ = Fn.bind(this, this.handleTextTrackAdd);
       this.handleTextTrackRemove_ = Fn.bind(this, this.handleTextTrackRemove);
+      this.removeOldTextTracks_ = Fn.bind(this, this.removeOldTextTracks);
```

Comment:
Why is this `Fn.bind` dance here anyway btw? I'm wondering if it is needed for `removeOldTextTracks_`?